clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_utils.py

```diff
@@ -197,8 +197,16 @@ def init_input_node(t1w, dwi, bvec, bval, dwi_json):
     check_dwi_volume(dwi, bvec, bval)
 
     # Read metadata from DWI JSON file:
-    [total_readout_time, phase_encoding_direction] = extract_metadata_from_json(
-        dwi_json, ["TotalReadoutTime", "PhaseEncodingDirection"]
+    [total_readout_time, phase_encoding_direction] = extract_metadata_from_dwi_json(
+        dwi_json,
+        [
+            "TotalReadoutTime",
+            "EstimatedTotalReadoutTime",
```

Comment:
This is very surprising when reading the code: you ask for 6 items and you only get two at the end...